website/docs/providers/type/community-index.html.markdown

```diff
@@ -90,7 +91,9 @@ please fill out this [community providers form](https://docs.google.com/forms/d/
 - [GoCD](https://github.com/beamly/terraform-provider-gocd)
 - [Google Calendar](https://github.com/sethvargo/terraform-provider-googlecalendar)
 - [Google G Suite](https://github.com/DeviaVir/terraform-provider-gsuite)
+- [Google Gmail Filter](https://github.com/yamamoto-febc/terraform-provider-gmailfilter)
 - [GorillaStack](https://github.com/GorillaStack/terraform-provider-gorillastack)
+- [Gmail](https://github.com/ndrpnt/terraform-provider-gmail/)
```

Comment:
@cgriggs01 not sure if you want this to be above GoCD to be in alphabetical order?

Reply:
Thanks aareet!